.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: CI Tests
```

Comment:
let's rename this consistently with the main Jaeger repo: Unit Tests / unit-tests / unit-tests.yml

Reply:
Sure. I will do as part of crossdock PR. Thank you.